src/module.ts

```diff
@@ -121,7 +121,6 @@ export default defineNuxtModule<ModuleOptions>({
   },
   setup(options, nuxt) {
     const { resolve, resolvePath } = createResolver(import.meta.url)
-    const resolveRuntimeModule = (path: string) => resolveModule(path, { paths: resolve('./runtime') })
```

Comment:
This was throwing an error and I am not even sure why it uses resolveModule instead of just resolve? Please check and confirm I did not miss anything crucial

Reply:
Legacy code from the old version of nuxt/kit 👍